airflow/auth/managers/fab/fab_auth_manager.py

```diff
@@ -209,7 +209,9 @@ def is_authorized_dag(
             resource_type = self._get_fab_resource_type(access_entity)
 
             if method == "GET":
-                return self._is_authorized(method=method, resource_type=resource_type, user=user)
+                return self._is_authorized_dag(
+                    method="GET", details=details, user=user
+                ) and self._is_authorized(method=method, resource_type=resource_type, user=user)
```

Comment:
```suggestion
                if not self._is_authorized_dag(method="GET", details=details, user=user):
                    return False
                return self._is_authorized(method=method, resource_type=resource_type, user=user)
```

Let’s break this, the statement is quite long.

Reply:
Done